From f87c0cfdd131e5c2a0849f45c8889ddeb9723de3 Mon Sep 17 00:00:00 2001
From: "Reddy, Alavala Srinivasa" <alavala.srinivasa.reddy@intel.com>
Date: Fri, 26 Aug 2022 21:26:52 +0530
Subject: [PATCH] Update Security patch level

The security patch level need to be updated
when ASB patches are merged.

Signed-off-by: Reddy, Alavala Srinivasa <alavala.srinivasa.reddy@intel.com>
---
 core/version_defaults.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index 65957c8859..5883925c73 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -103,7 +103,7 @@ ifndef PLATFORM_SECURITY_PATCH
     #  It must be of the form "YYYY-MM-DD" on production devices.
     #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
     #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-    PLATFORM_SECURITY_PATCH := 2022-08-05
+    PLATFORM_SECURITY_PATCH := 2022-09-01
 endif
 
 include $(BUILD_SYSTEM)/version_util.mk
-- 
2.17.1

